content/knowledge/flare_architecure.md

```diff
@@ -0,0 +1,152 @@
+* Every Flare validator independently verifies an underlying chain's state: If your Flare validator observes the canonical state of an underlying chain, then you will not lose safety against that chain.
+
+### Design
+
+State Connector is built into the validators.
+These validators (or validation nodes) are tasked with ordering transactions and confirming validity for consensus plus digesting state changes from integrated blockchains.
+Validators on the Flare Network operate on the ecosystem as well as the integrated blockchains’ ecosystems.
+
+The validation process consists of two stages:
+
+* congestion control by agreement on data availability
+* proving a transaction
+
+#### Congestion Control by Agreement on Data Availability
+
+In this stage, validators agree on the state of the underlying chain.
+This process of tracking state change availability occurs via a smart contract on the Ethereum Virtual Machine (EVM) level.
+The integrated blockchain validators can submit a valid claim availability proof in a competition to be first but only allows them to do so again after the timeout period has ended since the last valid claim availability proof.
+Additionally, a fee to submit the availability proofs is required to prevent attacking and spamming.
+The timeout period, fee, and count of finalized blocks from the integrated blockchains can all be changed via governance.
+
+#### Proving a Transaction
+
+After enough validators agree on the state of the network (data availability proof), validators need to approve the transaction.
+Validators will agree/disagree on the validity of the transaction using the Federated Byzantine Agreement for the network to reach a consensus.
+Once these criteria are met and that data is available, then the stored transactions in the storage contract are now available to be referenced by any other contract on the Flare Network.
+
+## Flare Time Series Oracle (FTSO)
+
+The FTSO provides externally sourced data estimates to the Flare Network in a decentralized manner.
+It does so by leveraging the distributed nature of the network and its participants.
+It generates rewards for active network participants and holders of Spark (FLR) via Signal Providers and F-Asset holders.
+
+The FTSO gets external data from Signal Providers (or Oracles) in the blockchain world.
+Signal Providers exist in the off-chain to collect data, such as asset prices, and deliver it to the FTSO.
+Signal providers on the Flare Network are responsible for delivering continuous and accurate data estimates to the FTSO to represent off-chain values on the Flare Network.
+Signal providers can provide their F-Asset and FLR votes to do this, but they also can receive delegated votes from other parties to perform this service on their behalf.
+Signal providers may take fees for this service.
```

Comment:
Let's be clear that signal providers receive a reward for accurately voting on prices, as determined by the algorithm. The top and bottom 25% of prices are discarded, and the average of the remaining is used. The 50% of price providers who are closest than receive rewards for their own tokens plus the percentage of fees, as defined by them on the smart contract. The default is 20%, and they can change it in 5% increments/decrements.